### PR TITLE
[RSDK-9165] Stop sessions when unimplemented received

### PR DESCRIPTION
**Pull request by njooma**

### CODE REVIEW

File: src/robot/session-manager.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default class SessionManager {
 
   private get client() {
     const transport = this.deferredTransport();
-    return createPromiseClient(RobotService, transport);
+    return createClient(RobotService, transport);
```
Comment (ethanlookpotts) [inline src/robot/session-manager.ts:35]: Why the change to `createClient`?

**Review by ethanlookpotts — APPROVED** [review]

**Review by purplenicole730 — APPROVED** [review]

**Review by stuqdog — APPROVED** [review]

File: src/robot/session-manager.ts (hunk 1 above)
Comment (njooma) [reply src/robot/session-manager.ts:35]: `createPromiseClient` is deprecated in favor of plain old `createClient`
